src/js/d3_svg_cell_renderer.js

```diff
@@ -0,0 +1,207 @@
+var utils = require('./utils');
```

Comment:
Here are a few sources in favor of the filename matching the exported variable:

http://stackoverflow.com/a/7273352/738484
https://github.com/felixge/node-style-guide/issues/30#issuecomment-61816649